components/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Heading from './Heading';
+
+describe('Heading component', () => {
+  test('renders the correct heading level with default props', () => {
+    render(<Heading>Test Heading</Heading>);
```

Comment:
I would force the `as` attribute anytime you use Heading. Explicitly make the implementor choose a heading size.

Reply:
Added in the `as` attribute